test/e2e/antreapolicy_test.go

```diff
@@ -4052,12 +4053,12 @@ func testACNPMulticastEgress(t *testing.T, data *TestData, acnpName, caseName, g
 
 // the matchers parameter is a list of regular expressions which will be matched against the
 // contents of the audit logs. The call will "succeed" if all matches are successful.
-func checkAuditLoggingResult(t *testing.T, data *TestData, nodeName, logLocator string, matchers []*regexp.Regexp) {
+func checkAuditLoggingResult(t *testing.T, data *TestData, nodeName, logFileDir, logLocator string, matchers []*regexp.Regexp) {
```

Comment:
I think `logFileDir` is misnamed? It is not actually a directory since you use it for the `cat` command below.

Reply:
Since we need a new check log func, I reverted this related change.